ext/liquid_c/tag_markup.c

```diff
@@ -65,6 +65,36 @@ void tag_markup_set_block_body(VALUE self, VALUE block_body_obj, block_body_t *b
     tag->block_body = block_body;
 }
 
+tag_markup_header_t *tag_markup_get_next_tag(document_body_entry_t *entry, tag_markup_header_t *current_tag)
```

Comment:
It doesn't seem like we should even need the entry pointer to get the next tag, since the next tag is immediately after the current one in memory.  Also, it seems like this function would be simpler if it didn't have to handle getting the first tag.

Suggested change: https://github.com/Shopify/liquid-c/commit/a46feef01ca7183ff293bc997c694042d1636375

Reply:
Fixed in deb3c0acb98a0c8ecf82d1086ad61dfc2ba251ee